recipes/epics-base/meta.yaml

```diff
@@ -0,0 +1,40 @@
+{% set version = "R3.14.12.4" %}
+package:
+    name: epics-base
+    version: {{ version }}
+
+source:
+    fn: base{{ version }}.tar.gz
+    url: http://www.aps.anl.gov/epics/download/base/base{{ version }}.tar.gz
+    sha256: 0ea4a74e83ff73d68b64de89bedf7c42102edc2cc69c69fd49eb35c04dc9e3de
+    patches:
+        - config_site.diff
+        - config_common.diff
+
+build:
+  number: 0
+  skip: True  # [not linux]
+
+requirements:
+  build:
+    - readline
```

Comment:
If I pin to 6.2*, I pick up the readline from the `defaults` channel since it is 6.2-2 and the one on conda-forge is 6.2-0.  Should conda-forge bump its build number?